.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,44 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
```

Comment:
I think this would be a great opportunity to use [issues forms ](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms) instead of templates because it forces the person reporting the bug to actually fill out certain fields.

Here's an example: https://github.com/remix-run/remix/blob/main/.github/ISSUE_TEMPLATE/bug_report.yml


If you prefer not to go that route, then a nitpick of mine for this would be to use heading for the titles instead of bold (i.e.: `#### Expected behavior` vs `**Expected behavior**`.

Reply:
Im down with that. I didnt know that was a thing